src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -602,6 +626,9 @@ TEST_CASE_VERSIONS("basic contract invocation", "[tx][soroban]")
             expectedRefund = spec.getResourceFee() - nonRefundableFee;
         }
 
+        // Starting from parallel soroban, the refund meta changes will be in
```

Comment:
I don't understand this comment, hasn't this been fixed above?

Reply:
Ah yeah this comment isn't accurate. It was written as I was debating how to deal with the tests. Removed.